qiskit/tools/wigner_function_visualisation.py

```diff
@@ -0,0 +1,264 @@
+"""
+
+Wigner Function Visualization Module
+
+Description:
+    This module contains functions for calculating and plotting the Wigner
+    function. The ways the Wigner function can be calculated includes:
+    - The equal angle slice bloch sphere plot of an arbitrary state
+      described by a density matrix or state vector.
+    - An array of points in phase space to be plotted as a curve
+    - A single point in phase space
+    - An NxN array plotted as a plaquette
+
+References:
```

Comment:
You can move these reference into the function description for `plot_wigner_function` and/or `wigner_function` when moving into the visualization.py module